### PR TITLE
feat(RichTextEditor): alignment behaviour in lists

### PR DESCRIPTION
**Pull request by celine-s**
Tickets:
* [active state of toolbar buttons](https://app.clickup.com/t/8677k2p1f)
* [alignment behaviour](https://app.clickup.com/t/8677k2ncq)

### CODE REVIEW

**Review by Walkeezy — APPROVED** [review]
Works nicely and code looks good as well 👍🏻

File: src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx [hunk 1]
```diff
@@ -6,14 +6,22 @@ import { PlateRenderElementProps, TTodoListItemElement, getRootProps, setNodes }
 import { merge } from '@utilities/merge';
 import { MarkupElement } from '../MarkupElement';
 import { ELEMENT_CHECK_ITEM } from './id';
+import { justifyClassNames } from '../TextStylePlugin/TextStyles/alignment';
 
 export const CheckboxListElementNode = (props: PlateRenderElementProps) => {
     const { attributes, children, nodeProps, element, editor } = props;
     const rootProps = getRootProps(props);
     const checked = element.checked as boolean;
+    const align = (element.align as unknown as string) ?? 'left';
```
Comment (triggertoo) [inline src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx:15]:
```suggestion
    const align = (element.align as string) ?? 'left';
```

File: src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx [hunk 2]
```diff
@@ -19,14 +22,25 @@ export const getTextStyle = (styledNode: TNode) => {
...
+                getColumnBreakClasses(element),
+                align ? justifyClassNames[align] : 'tw-justify-start tw-flex',
+            ])}
             {...attributes}
         >
-            {children}
+            <span
+                className={merge([
+                    isEmpty && 'tw-w-4',
+                    designTokens[getTextStyle(element.children[0])]?.textDecoration === 'underline' && 'tw-underline',
```
Comment (triggertoo) [inline src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx:39]: is it necessary that the underline is applied on the LIC level? how do we deal with other styles?

File: src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx (hunk 1 above)
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/CheckboxListPlugin/CheckboxListElement.tsx]: adapted 👍

File: src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx (hunk 2 above)
Comment (celine-s) [reply src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx:39]: underline was already before handled individually.

**Review by triggertoo — APPROVED** [review]
👍

**Pull request merged**